website/react-magma-docs/src/pages/api/tree-view.mdx

```diff
@@ -597,6 +597,68 @@ export function Example() {
 }
 ```
 
+## Check All Children / Check all parents when children are checked
+
+```typescript
+import React from 'react';
+import { Toggle } from 'react-magma-dom';
```

Comment:
Missing a few imports:
```suggestion
import { TreeView, TreeItem, TreeViewSelectable, Toggle } from 'react-magma-dom';
```

Reply:
Somewhat interesting fact - these imports seem to be shared between all the sandboxes on page

Reply:
The problem is that when a user tries to click on the codesandbox link from the docs site to view a single example, if those imports aren't there, the example doesn't load as expected.